package.json

```diff
@@ -46,11 +46,13 @@
   },
 
   "scripts": {
-	"test": "mocha tests.coffee --compilers coffee:coffee-script/register"
+    "test": "mocha test/dice_test.coffee --compilers coffee:coffee-script/register"
```

Comment:
This should stay as it was, and `tests.coffee` should load `test/*`, or the command should search for `tests/*` and include them on the command line.